synthtool/gcp/templates/node_library/.kokoro/continuous/node10/publish-docs.cfg

```diff
@@ -0,0 +1,4 @@
+env_vars: {
```

Comment:
Didn't mean to add these kokoro jobs to continuous/ and presubmit/, they'll  be removed in another commit.